fastlane/jetpack_metadata/android/es-ES/changelogs/1064.txt

```diff
@@ -0,0 +1,6 @@
+17.6:
+Con el bloque Audio, puedes añadir podcasts, música y archivos de audio a tus entradas y páginas. También está disponible para usuarios del plan gratuito.
+
+Utiliza el nuevo botón "Establecer como destacado" del bloque Imagen para convertir cualquier imagen en una imagen *destacada*.
+
+Por último, dejamos claro que no es posible editar la página Archivos de entradas, que se genera automáticamente.
```

Comment:
Fun fact: the `en-ES` translators decided to translate slightly differently the release notes used for WordPress from the ones used for Jetpack (this is true for 3-4 JP locales here not just `es-ES`), despite those having the exact same source/English copy for this version. So basically instead of copy/pasting the translation that they already did from WP's release notes GlotPress project to reuse them for JP's release notes in GP, they re-translated them leading to slight differences in wording.

The good news about that is that it allows us to confirm that we pulled those translations from the right GlotPress project (JP) into the right files (`jetpack_metadata/*`) since we can differentiate them from WP's ones 😛